### ISSUE TITLE
Tensor Imaging (Notebook 3)

### ISSUE THREAD

**Issue by kaitj** [carpentries-incubator/SDC-BIDS-dMRI#12]
Looking to also update Notebook 3 on Diffusion Tensor Imaging (DTI). This notebook should highlight what DTI is, why its important and the pros and cons.

Main things to update;
- [x] Update with images / processing from ds000221
- [x] Better explain different quantitative metrics (e.g. FA, MD, AD, RD)

A separate notebook to be included for higher order models / more current models used in research.

**Comment by jhlegarreta** [comment]
@kaitj my current bandwidth is limited, but if the task is not due immediately, I would be pleased to work with you on this.

**Comment by kaitj** [comment]
@jhlegarreta That sounds great! Definitely not an immediate due date! We are planning on storing the processed data on OSF (https://osf.io/cmq8a/). At this point, we've only selected 10 random subjects to use!

**Comment by jhlegarreta** [comment]
@kaitj maybe at some point having a short meeting would be useful for me to start knowing better the ecosystem of tools required and where I could specifically start to contribute incrementally. We could you Slack whenever you have time for this.

**Comment by kaitj** [comment]
@jhlegarreta Sure! I am free most days (except Wednesdays). Let's set something up and we can chat about it!

**Comment by jhlegarreta** [comment]
@kaitj Sounds good. Maybe it will be easier to set up a date and talk if we chat through Slack? You can find me by my name or username in the neuroimaging Slack channel !

**Comment by jhlegarreta** [comment]
@kaitj the following item:
```
Better explain different quantitative metrics (e.g. FA, MD, AD, RD)
```
was addressed in PR #18, so I guess it can be marked as done in the issue.

**Comment by kaitj** [comment]
Closing this issue. Thought I had linked #64 to this previously.
